p5.easycam.js

```diff
@@ -605,7 +605,7 @@ class EasyCam {
       this.camLAT = this.getCenter  (this.camLAT);
       this.camRUP = this.getUpVector(this.camRUP);
       
-      renderer.camera(this.camEYE[0], this.camEYE[1], this.camEYE[2],
+      renderer._curCamera.camera(this.camEYE[0], this.camEYE[1], this.camEYE[2],
```

Comment:
Would it be worth doing something  like:

```
var currentCam = renderer._curCamera || renderer;
currentCam.camera(this.camEYE[0], this.camEYE[1], this.camEYE[2],
```

So there's backwards compatibility?

Reply:
This repo appears to be abandoned, thus this PR will likely never be merged. The newer code base is [here](https://github.com/freshfork/p5.EasyCam) and it is backwards compatible. Good suggestion!

Reply:
Ah, brilliant, thanks for pointing me to that. I'll switch to using that version